src/main/java/org/sagebionetworks/template/utils/ArtifactDownloadImpl.java

```diff
@@ -54,7 +54,7 @@ public File downloadFile(String url) {
 	}
 
 	@Override
-	public File downloadFileFromZip(String url, String filename) {
+	public File downloadFileFromZip(String url, String filePath) {
```

Comment:
I was expecting that we would be extracting multiple files from the same project zip.  Is the current plan to re-download and extract each file from the zip?